derivation.nix

```diff
@@ -26,6 +26,8 @@ in
     , distDir ? "dist"
     , installInPlace ? false
     , installEnv ? { }
+    , installAndBuildEnv ? { }
```

Comment:
My preference would be to not have this shared env argument. If the user is going to de-duplicate their ENV attrsets to use them here they could just as easily do `sharedEnv // installEnv` and `sharedEnv // buildEnv` on the call side.

Reply:
Fair point. That does simplify the interface.

Reply:
Done.